src/main/kotlin/com/cognifide/gradle/aem/pkg/ComposeTask.kt

```diff
@@ -172,27 +173,31 @@ open class ComposeTask : Zip(), AemTask {
         contentCollectors.onEach { it() }
     }
 
+    fun includeProject(projectPath: String) {
+        includeProject(project.findProject(projectPath))
+    }
+
+    fun includeProject(project: Project) {
+        includeContent(project)
+        includeBundles(project)
+    }
+
     fun includeSubprojects() {
-        includeSubprojects(true)
+        includeProjects("${project.path}:*")
     }
 
-    fun includeSubprojects(withSamePathPrefix: Boolean) {
+    fun includeProjects(pathFilter: String) {
         project.gradle.afterProject { subproject ->
-            if (subproject.path != project.path && subproject.plugins.hasPlugin(AemPlugin.ID)) {
-                if (!withSamePathPrefix || subproject.path.startsWith("${project.path}:")) {
```

Comment:
this is invoked in configuration phase, so that when assembly is being created, this does not tell about real copying being executed in CopyTask.execute()

I decided to just remove that info, to prevent confusing situations while debugging.